### PR TITLE
feat: [SEI-8521]: Add new SEI icons

### PR DESCRIPTION
**Pull request by sharath-levelops**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2669

File: packages/icons/src/sei-integration-map.svg
[the remark this answers is not in the document]
Comment (knagurski) [reply packages/icons/src/sei-integration-map.svg:1]: If we add a few more, we can get a full jigsaw going on lol

**Comment by sharath-levelops** [comment]
retrigger typecheck

**Comment by sharath-levelops** [comment]
retrigger titlecheck

**Review by thanh-levelops — APPROVED** [review]

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2671

**Pull request merged**